encoder/libav_encoder.cpp

```diff
@@ -405,12 +442,12 @@ void LibAvEncoder::initOutput()
 	if (!(out_fmt_ctx_->flags & AVFMT_NOFILE))
 	{
 		std::string filename = options_->output;
-		if (options_->segment){			
+		// if (options_->segment || options_->split){			
```

Comment:
That looks a bit strange to me, I'm a bit worried about the curly brace being commented out and whether there's a trailing brace somewhere...